modules/loader.js

```diff
@@ -310,7 +311,7 @@ __private.loadTransactions = function (cb) {
  * @implements {modules.blocks.deleteAfterBlock}
  * @implements {modules.blocks.loadLastBlock}
  * @emits exit
- * @throws {string} When fails to match genesis block with database
+ * @throws {string} When fails to match genesis block with database or rounds exceptions doesn't match database
```

Comment:
`On failure to match genesis block with database, or when rounds exceptions do not match database`